### PR TITLE
feat(webinar): add webinar related props

### PR DESCRIPTION
**Pull request by JudyZhuHz**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-480775

## This pull request addresses

JS SDK - Manager Webinar status

## by making the following changes

< DESCRIBE YOUR CHANGES >

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mickelr — APPROVED** [review]
LGTM

File: packages/@webex/plugin-meetings/src/webinar/collection.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) 2015-2023 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import AmpCollection from 'ampersand-collection';
```
Comment (arun3528) [inline packages/@webex/plugin-meetings/src/webinar/collection.ts:5]: why use ampersand collection here and not just a object map ?

Comment (arun3528) [reply packages/@webex/plugin-meetings/src/webinar/collection.ts:5]: its not being maintained

**Review by arun3528 — CHANGES_REQUESTED** [review]

File: packages/@webex/plugin-meetings/src/webinar/collection.ts (hunk 1 above)
Comment (arun3528) [reply packages/@webex/plugin-meetings/src/webinar/collection.ts:5]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Comment (JudyZhuHz) [reply packages/@webex/plugin-meetings/src/webinar/collection.ts]: @arun3528  Please help review it again, thanks

**Review by arun3528 — APPROVED** [review]

**Comment by arun3528** [comment]
i could just do new Map() which has get and set but this is fine too

**Pull request merged**